docs/developer-guide/plugins-scenarios.md

```diff
@@ -7,10 +7,10 @@
 Transforming a request object synchronously is trivial. By "synchronously" we mean in the 
 same thread, in non-blocking fashion. 
 
-Just call `request.newBuilder()` to
-create a new `HttpRequest.Builder` object. It has already been initialised with the copy of
-the original `request`. Modify the builder as desired, construct a new version, and pass
-it to the `chain.proceed()`, as the example demonstrates:
+Just call `request.newBuilder()` to create a new `HttpRequest.Builder` object. 
+It has already been initialised with the copy of the original `request`. Modify 
+the builder as desired, construct a new version, and pass it to the `chain.proceed()`, 
```

Comment:
I don't see any differences here or most of the text changes (other than new lines), so it's a bit difficult to find what's actually changed.  Maybe you can say  "pass it to 'chain.proceed' " removing the 'the'

Reply:
The lines were really too long to meaningfully edit them. So I had to add line breaks ;-(

There wasn't any text changes as such. Only an occasional change of `Observable` to `StyxObservable` etc.